### PR TITLE
The New York Times - Article Clone

### PR DESCRIPTION
**Pull request by tadeuasarro**
Veryfing Linters.

### CODE REVIEW

**Comment by tadeuasarro** [comment]
# Article

The project has been finished.
Linters were used to verify the code.
README.md updated.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+                    <p class="main-text">
+                        Nulla ut rutrum nisi. Sed quis viverra odio, eu eleifend turpis. In at condimentum elit. Aliquam egestas aliquet diam sit amet malesuada. Nunc vehicula bibendum odio, ut molestie nunc. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Donec eu ipsum massa. Duis nisi massa, pharetra id lectus viverra, fermentum iaculis justo. Vivamus fringilla ipsum sed nulla imperdiet, ac malesuada metus gravida.
+                    </p>
+                    <div class='image2Con'>
+                        <span><strong>Lorem ipsum dolor sit amet.</strong></span><br><br>
+                        <span><small>Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit beatae ab dicta accusamus possimus laudantium et suscipit rem aliquam, numquam blanditiis cupiditate! In, expedita! Porro, enim quisquam? Fugit, qui deserunt?</small></span>
+                        <img src="img/inflation-945.png" width='945'  alt="inflation picture">
+                        <hr class="image2-hr"><br>
+                        <span style='font-family:"Arial"'>Lorem ipsum dolor sit amet.</span>
```
Comment (HADDADSOHAIB) [inline index.html]: Remove inline style, your styles should be on the css files

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello @tadeuasarro  and @Alexoid1   :hand: 

### Good job, you did great job with the project :+1: 

### Unfortunately, there are still some changes to get the project approved:
- The pull request should look professional, please add a description to the pull request:
Screenshot from 2020-06-16 00-21-14
This is not enough, also the title could be improved (ex: Newyork article clone, article clone ....).
- The logo should be a link, also the subscribe now and log in buttons are different from the original:
Original project:
Screenshot from 2020-06-16 00-24-40
Your project:
Screenshot from 2020-06-16 00-26-04
- All social media icons should be links like the original:
Screenshot from 2020-06-16 00-27-46
- All the articles in the editor picks should be links:
Screenshot from 2020-06-16 00-30-04
- A lot of contents are missing from this part, the project is a clone, so it must be close to the original website, please add the missing contents:
Your project:
Screenshot from 2020-06-16 00-30-53
Original:
Screenshot from 2020-06-16 00-32-28
- All the elements in this section should be links:
Screenshot from 2020-06-16 00-33-50
also, try to improve the design, they're a lot of elements that need to be improved, for example:
Screenshot from 2020-06-16 00-36-21
Remove the duplicate top separater.
Screenshot from 2020-06-16 00-37-20
Add separated between these two sections.
- There is a lot of space, please remove that:
original:
Screenshot from 2020-06-16 00-38-12
your project:
Screenshot from 2020-06-16 00-38-05
- All these elements should be links:
Screenshot from 2020-06-16 00-39-36
- Please remove inline styling (check the comments).
- [Optional] Think of using a main tag for the dominant part of your HTML file.
- [Optional] Think of using a footer tag instead of a div.
### Once you are done with the changes, asked for another code review

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: index.html [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+                            <span class="image-2-span">EXPANSION Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.</span>
+                        </div>
+                        <span><strong>The Theory of Inflation</strong></span><br><br>
+                        <span><small>Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</small></span>
+                        <img class="image-2-actually" src="img/inflation-945.png" width='945'  alt="inflation picture">
+                        <hr class="image2-hr"><br>
+                        <span style='font-family:"Arial"'>By LARRY BUCHANAN and JONATHAN CORUM</span>
```
Comment (andresporras3423) [inline index.html]: Don't use inline styles

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,593 @@
...
+.right-elem a {
+  margin-right: 10px;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+}
+
+.icon-arrow i {
+  color: black;
+}
```
Comment (andresporras3423) [inline styles.css]: This selector has the same styles as the selector on line 102. Group selectors to minimize code.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,593 @@
...
+
+.container h2 {
+  margin: 30px 150px;
+  font-family: 'Balsamiq Sans', cursive;
+  font-size: 35px;
+}
+
+.image1 {
+  text-align: center;
+}
```
Comment (andresporras3423) [inline styles.css:127]: This selector has the same styles as the selector on line 163. Group selectors to minimize code.

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,593 @@
...
+  background-color: #eee;
+  align-items: center;
+  justify-content: center;
+  font-size: 12px;
+}
+
+.social a {
+  text-decoration: none;
+  color: black;
+}
```
Comment (andresporras3423) [inline styles.css:193]: This selector has the same styles as selectors on lines 266 and 268. Group selectors to minimize code.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,593 @@
...
+
+.floatAnchors2:hover {
+  color: grey;
+}
+
+.floatAnchors:hover {
+  text-decoration: underline;
+}
```
Comment (andresporras3423) [inline styles.css]: This selector has the same styles as the selector on line 591. Group selectors to minimize code.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @tadeuasarro @Alexoid1** 
Good job with the improvements to the project :muscle:  
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [ ] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).
- [ ] Don't use inline styles in your HTML
- [ ] content missing in a part of your page, check the original:



and your clone:



### OPTIONAL

- [ ] it's recommended to use the main tag for the dominant content of the body of the document
- [ ] it's recommended to use nav tag for the major navigation links

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by tadeuasarro** [comment]
# The final version of the New York Article Clone:

Linters were run;
The text in image 2 has been fixed;
Selectors have been grouped in styles.css;
Inline styles were removed from index.html;
Some elements' colors have been fixed.

**Review by elmejdki — APPROVED** [review]
# Great job @team :clap: :blush:, your project looks super like the original :tophat: 
### status: approved

your project was approved you can merge now to master.

### Happy Coding
_If you have any type of questions or concerns you want to discuss them with me don't hesitate to contact me on slack_

**Pull request merged**